shared/src/assets/icons/solAssets.tsx

```diff
@@ -4,61 +4,31 @@ type SVGProps = React.SVGAttributes<SVGElement>;
 
 export const SolanaLogo: React.FC<SVGProps> = (props) => (
   <svg
-    width="646"
-    height="96"
-    viewBox="0 0 646 96"
+    width="41"
+    height="32"
+    viewBox="0 0 41 32"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
+    style={props.style || { padding: 4 }}
     {...props}
   >
-    <g clipPath="url(#clip0_4_583)">
+    <g clip-path="url(#clip0_2204_947)">
```

Comment:
I think this one need to be `clipPath` if converted to react code, otherwise there will be warning